tests/pkcli/admin_test.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+u"""Test pkcli.admin
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import os
+import pytest
+
+def setup_module(module):
+    from sirepo import srunit
+    import os
+    srunit.setup_srdb_root()
+    os.environ.update(
+        SIREPO_FEATURE_CONFIG_DEFAULT_PROPRIETARY_SIM_TYPES='jupyterhublogin',
+    )
+
+
+def test_no_user():
+    import sirepo.pkcli.admin
+    from pykern import pkunit
+    with pkunit.pkexcept(AssertionError):
```

Comment:
That's probably fine but maybe want to test for a specific regex.

Reply:
7277606d2